packages/runner-shared/src/iframe/aut-iframe.js

```diff
@@ -91,6 +124,17 @@ export class AutIframe {
   }
 
   restoreDom = (snapshot) => {
+    if (!this.doesAUTMatchTopOriginPolicy()) {
+      this.$iframe.one('load', () => {
+        this.restoreDom(snapshot)
```

Comment:
a `load` event fires here when the src is removed. This is equivalent to `about:blank` according to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-src), but isn't obvious to the user. This doesn't resort in a log message being generated for a `new page`.

Reply:
Maybe add this as a code comment for posterity?

Reply:
added in [bf94d55](https://github.com/cypress-io/cypress/pull/20949/commits/bf94d55c7c86bf63e830f8d73994166b5a3a6478)